projects/storefrontapp-e2e-cypress/cypress/integration/mobile/my-account/order-history-no-orders-flow-mobile.e2e-spec.ts

```diff
@@ -14,6 +14,8 @@ describe(`${formats.mobile.width +
 
     loginHelper.registerUser();
 
+    loginHelper.loginUser();
```

Comment:
No need to login here and then logout. Replace it with wait for alert: `alertHelpers.getAlert();`
Remove `waitForHomePage` and sign out.

Reply:
We have to discuss this one